frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -47,6 +48,11 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
       </Bullseye>
     );
   }
+  const handleVersionUpdate = async (updatePromise: Promise<unknown>): Promise<void> => {
+    await updatePromise;
```

Comment:
You're not calling this, you're missing the `()` in `updatePromise()`

Reply:
Ah - I was wrong here, you're passing in an already-running promise and you just need to await it. My mistake!